plugins/catalog-backend/src/processing/DefaultCatalogProcessingEngine.ts

```diff
@@ -35,9 +35,16 @@ import { Stitcher } from '../stitching/Stitcher';
 import { startTaskPipeline } from './TaskPipeline';
 import { PluginTaskScheduler } from '@backstage/backend-tasks';
 import { Config } from '@backstage/config';
+import {
+  addEntityAttributes,
+  TRACER_ID,
+  withActiveSpan,
+} from '../util/opentelemetry';
 
 const CACHE_TTL = 5;
 
+const tracer = trace.getTracer(TRACER_ID);
```

Comment:
Is there any particular reason to not put this at the top of `startPipeline`? Or even at the top of `processTask`? If want to you replace the tracer, you are now strongly dependent on module import / initialization order which may be unfortunate.